optimum/exporters/tasks.py

```diff
@@ -512,6 +516,14 @@ class TasksManager:
             "seq2seq-lm-with-past",
             onnx="T5OnnxConfig",
         ),
+        "unet": supported_tasks_mapping(
```

Comment:
Just to understand: what is the motive for having different parts of the model in different configs? People are interested in exporting different parts standalone?

Reply:
Currently we export each model's component separately (U-NET, text encoder and VAE) in order for users to be able to different schedulers as well as to use the resulting exported model for different tasks. Also this integration will make it possible to run inference with `diffusers.OnnxStableDiffusionPipeline`.  It would be great to extend this work in future PRs

Reply:
So this means that with this PR I would be able to export the unet part alone, if I want to? How does it work with exporters since different parts of the model are in subfolders?

For example:

```
python -m optimum.exporters.onnx -m https://huggingface.co/runwayml/stable-diffusion-v1-5 out_onnx/
```

will export all the subparts.

But given that `unet` and such are added in the TasksManager, I would expect that this means I am able to use the exporters CLI to export only part of the model. I don't think this would be possible currently since parts of the models are stored in subfolders, e.g. https://huggingface.co/runwayml/stable-diffusion-v1-5/tree/main

So I'm not sure it is good to add those here.

Reply:
To export one specific component or a subset of them, you should have your adapted `fn_get_models_from_config` but I feel this is a very specific case that we can tackle in a future PR in case it's requested by users

Reply:
I think this is not a big deal for now, what we can do is add a `TODO` comment in the code for now. Maybe also create an issue to remember to work on it later.

Reply:
What would be the place to list all supported model types and modalities supported in the exporters ONNX CLI then?

I understand this is not a big deal, I just think it is good to have a way to know what is supported and what is not. This change is breaking this.

Reply:
Like `"seq2seq-lm"` will load the model using the class `AutoModelForSeq2SeqLM`, `"stable-diffusion"` will use `StableDiffusionPipeline` to load the model's different component. I don't think `"text-to-image"` is adapted here because this task include other types of models which should not be loaded with `StableDiffusionPipeline` and also exclude other tasks such as `image-to-image`.

Reply:
The difference is that arguably `seq2seq-lm` is a task while `stable-diffusion` is a model architecture. Stable diffusion is not a task.

Reply:
agreed!

Reply:
I think @fxmarty makes a good point, but to not make this PR last to long, I think it would be perfect to create issues for that so we can work on them later.

Reply:
Created an issue => #596